tests/lib/rules/jsx-no-target-blank.js

```diff
@@ -96,60 +96,97 @@ ruleTester.run('jsx-no-target-blank', rule, {
+    output: '<a target="_blank" rel="noreferrer" href="http://example.com"></a>',
     errors: defaultErrors
   }, {
     code: '<a target="_blank" rel="" href="http://example.com"></a>',
+    output: '<a target="_blank" rel="noreferrer" href="http://example.com"></a>',
+    errors: defaultErrors
+  }, {
+    code: '<a target="_blank" rel={0} href="http://example.com"></a>',
+    output: '<a target="_blank" rel={"noreferrer"} href="http://example.com"></a>',
+    errors: defaultErrors
+  }, {
+    code: '<a target="_blank" rel={false} href="http://example.com"></a>',
+    output: '<a target="_blank" rel={"noreferrer"} href="http://example.com"></a>',
+    errors: defaultErrors
+  }, {
+    code: '<a target="_blank" rel={null} href="http://example.com"></a>',
+    output: '<a target="_blank" rel={"noreferrer"} href="http://example.com"></a>',
     errors: defaultErrors
   }, {
     code: '<a target="_blank" rel="noopenernoreferrer" href="http://example.com"></a>',
+    output: '<a target="_blank" rel="noopener noreferrer" href="http://example.com"></a>',
     errors: defaultErrors
   }, {
     code: '<a target="_blank" rel="no referrer" href="http://example.com"></a>',
+    output: '<a target="_blank" rel="no referrer noreferrer" href="http://example.com"></a>',
     errors: defaultErrors
   }, {
     code: '<a target="_BLANK" href="http://example.com"></a>',
+    output: '<a target="_BLANK" rel="noreferrer" href="http://example.com"></a>',
     errors: defaultErrors
   }, {
     code: '<a target="_blank" href="//example.com"></a>',
+    output: '<a target="_blank" rel="noreferrer" href="//example.com"></a>',
     errors: defaultErrors
   }, {
     code: '<a target="_blank" href="//example.com" rel={true}></a>',
+    output: '<a target="_blank" href="//example.com" rel={"noreferrer"}></a>',
     errors: defaultErrors
   }, {
     code: '<a target="_blank" href="//example.com" rel={3}></a>',
+    output: '<a target="_blank" href="//example.com" rel={"noreferrer"}></a>',
     errors: defaultErrors
   }, {
     code: '<a target="_blank" href="//example.com" rel={null}></a>',
+    output: '<a target="_blank" href="//example.com" rel={"noreferrer"}></a>',
+    errors: defaultErrors
+  }, {
+    code: '<a target="_blank" href="//example.com" rel={getRel()}></a>',
     errors: defaultErrors
   }, {
     code: '<a target="_blank" href="//example.com" rel={"noopenernoreferrer"}></a>',
+    output: '<a target="_blank" href="//example.com" rel={"noopener noreferrer"}></a>',
     errors: defaultErrors
   }, {
     code: '<a target={"_blank"} href={"//example.com"} rel={"noopenernoreferrer"}></a>',
+    output: '<a target={"_blank"} href={"//example.com"} rel={"noopener noreferrer"}></a>',
+    errors: defaultErrors
+  }, {
+    code: '<a target={"_blank"} href={"//example.com"} rel={"noopenernoreferrernoreferrernoreferrernoreferrernoreferrer"}></a>',
+    output: '<a target={"_blank"} href={"//example.com"} rel={"noopener noreferrer"}></a>',
```

Comment:
it's fine to keep it here since the original was both a string and had curly braces